packages/core/src/validate.ts

```diff
@@ -196,7 +247,7 @@ const errorInfo: ErrorDescriptions<ValidationError> = {
   },
   'external-library-linking': {
     msg: e => `Linking external libraries like \`${e.name}\` is not yet supported`,
-    hint: `Stick to libraries with internal functions only`,
+    hint: `Use libraries with internal functions only, or skip this check with the \`unsafeAllowLinkedLibraries\` flag if you have manually checked that the libraries are upgrade safe`,
```

Comment:
This hint is rather long and it doesn't look very good in the console. Options: 1) insert <code>\n____</code> in the middle to wrap it manually (underscores should be spaces for indentation), 2) handle it in `describeError`.

Reply:
Done. Implemented option 1. This is the approach also used in the Warning messages which are quite long.